bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd

```diff
@@ -1,57 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" 
```

Comment:
Yes - I mentioned that I think at the end of the last commit. I'm using the ESH formatter - I do't know if it covers XML or not, or if there's anything special I need to do, but ESH formatter is selected...